dproc/dproc_test.go

```diff
@@ -0,0 +1,102 @@
+package dproc
+
+import (
+	"fmt"
+	"path/filepath"
+	"testing"
+
+	"github.com/cloudspannerecosystem/harbourbridge/profiles"
+	"github.com/stretchr/testify/assert"
+)
+
+// TODO: add unit tests for otjer methods in dproc package
```

Comment:
Please add the unit tests along with the PR

Reply:
Added more test cases for `GetDataprocRequestParams()`, 

Didn't add any unit tests for others methods (`CreateDataprocBatchClient()`, `TriggerDataprocTemplate()`), these two methods are used used for only for making dataproc API calls to create dataproc client and  to submit the job via generated params from `GetDataprocRequestParams()` , to have unit tests for these two methods will need a lot of mocking, there won't be much to test after mocking, Plan is to test it in integration test.

Let me know what you think.

Commit: https://github.com/cloudspannerecosystem/harbourbridge/pull/520/commits/64946605e2140447fbb83f6bc9b323e8d4a59d4b